libsrc/target/tiki100/gr_setpalette.asm

```diff
@@ -63,39 +65,42 @@ set_loop:
 ;
 ;
 ; Input:
-; 	A = Palette
-; 	B = Position
-;	D = number of colours
+; 	A = Palette color
+; 	B = Palette index
+;	D = number of colours in selected mode
 ;
 .do_set
 	cpl
-        LD E,A
+	LD E,A
 	ld	hl,$f04d
 .palette_loop
-        PUSH DE
-        LD A,E
-        DI
-        OUT ($14),A             ; Palette register (prepare the color to be loaded)
-        OUT ($14),A             ; Palette register (do it again to be sure)
-	ld	a,(hl)
+
+	PUSH	DE
+	LD	A,E
+	DI
+	OUT	($14),A	     ; Palette register (prepare the color to be loaded)
+	ld	a,(PORT_0C_COPY)
```

Comment:
I feel like I'm missing something - why not just replace the hl constant in the original with PORT_0C_COPY and save a byte on the new code?

Reply:
Forgot to add it, will do

Reply:
The idea was really just to use the constant instead of a value directly, and eventually make the code more readable. When you mention it, using hl does indeed save a byte and a clock-cycle.